src/internal/components/live-region/index.tsx

```diff
@@ -93,13 +94,17 @@ function LiveRegion({ assertive = false, delay = 10, children, ...restProps }: L
   });
 
   return (
-    <ScreenreaderOnly {...restProps} className={clsx(styles.root, restProps.className)}>
-      <span aria-hidden="true">
-        <span ref={sourceRef}>{children}</span>
-      </span>
+    <>
+      {visible && children}
 
-      <span ref={targetRef} aria-atomic="true" aria-live={assertive ? 'assertive' : 'polite'}></span>
-    </ScreenreaderOnly>
+      <ScreenreaderOnly {...restProps} className={clsx(styles.root, restProps.className)}>
```

Comment:
Why not something like this?

```
const Wrapper = visible ? 'span' : ScreenreaderOnly;

return <ScreenreaderOnly>...</ScreenreaderOnly>
```

having double text may cause problems when copying it, for example